src/arrays.js

```diff
@@ -1,59 +1,164 @@
 export const getNthElement = (index, array) => {
-  // your code here
+  
+if(index >= array.length) {
+  
+  	return array[index%(array.length)]
+}
+  else {
```

Comment:
Because you already return out of the function if `index >= array.length`, you don't need the `else` statement - you can simply do:
```js
if (index >= array.length) {
  return array[index % array.length]
}

return array[index]
```